src/main/workspace/tree/visitor/OrderTreeVisitor.ts

```
@@ -0,0 +1,25 @@
+import { Visitor } from './Visitor';
+import Folder from '../Folder';
+import File from '../File';
+// Tiny helper visitor: "where does this node rank in the folder-first ordering?"
+// Driven by double dispatch — no getType() calls.
+const folderFirst: Visitor<number> = {
+  VisitFolder: () => 0,
+  VisitFile:   () => 1,
```

ESLint

[error] Extra space before value for key 'VisitFile'.

(key-spacing)